Development/src/pages/devices/DevicesShow.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import {
+    ArrayField,
+    Button,
+    Datagrid,
+    FunctionField,
+    ListButton,
+    ReferenceArrayField,
+    ReferenceField,
+    ReferenceManyField,
+    ShowView,
+    SimpleShowLayout,
+    SingleFieldList,
+    TextField,
+    TopToolbar,
+    useShowController,
+} from 'react-admin';
+import Cookies from 'universal-cookie';
+import JsonIcon from '../../components/JsonIcon';
+import MapTags from '../../components/TagsField';
+import TAIField from '../../components/TAIField';
+import UrlField from '../../components/URLField';
+import QueryVersion from '../../components/QueryVersion';
+import ChipConditionalLabel from '../../components/ChipConditionalLabel';
+
+const cookies = new Cookies();
+
+const DevicesTitle = ({ record }) => {
```

Comment:
May be worth refactoring ResourceTitle into it's own component

Reply:
We realised that it's a bit awkward to get the resource type, so leaving this for a future PR.